docs/content/users/install/ddev-installation.md

```diff
@@ -2,50 +2,44 @@
 
 Once you’ve [installed a Docker provider](docker-installation.md), you’re ready to install DDEV!
 
-Installing and upgrading DDEV are nearly the same thing, because you're upgrading the `ddev` binary that talks with Docker. You can update this file like other software on your system, whether it’s with a package manager or traditional installer.
-
 === "macOS"
 
     ## macOS
 
     ### Homebrew
 
-    We recommend [Homebrew](https://brew.sh/) because it’s the easiest and most reliable way to install and upgrade DDEV:
+    [Homebrew](https://brew.sh/) is the easiest and most reliable way to install and upgrade DDEV:
 
     ```bash
+    # Install DDEV
     brew install ddev/ddev/ddev
-    ```
-
-    ```bash
-    brew upgrade ddev
-    ```
-
-    As a one-time initialization, run
 
-    ```bash
+    # Initialize mkcert
     mkcert -install
+
+    # Optionally enable Mutagen for the best performance (recommended!)
+    ddev config global --mutagen-enabled
     ```
 
     ### Install Script
 
-    !!!tip
-        The install script works on macOS, Linux, and Windows WSL2.
+    The [install script](https://github.com/ddev/ddev/blob/master/scripts/install_ddev.sh) is another option. It downloads, verifies, and sets up the `ddev` binary:
 
-    The [install script](https://github.com/ddev/ddev/blob/master/scripts/install_ddev.sh) is an alternate way to install or upgrade DDEV. It downloads, verifies, and sets up the `ddev` binary.
-
-    To install or update DDEV:
-
-    ```
+    ```bash
+    # Download and run the install script
     curl -fsSL https://ddev.com/install.sh | bash
-    ```
-
-    You can include a `-s <version>` argument to install a specific release or a prerelease version:
 
-    ```
-    curl -fsSL https://ddev.com/install.sh | bash -s v1.21.4
+    # Optionally enable Mutagen for the best performance (recommended!)
```

Comment:
It looks like this mutagen statement got migrated to just the install script section?

Reply:
🤔 I’m not following. It [currently](https://ddev.readthedocs.io/en/stable/users/install/ddev-installation/#install-script) looks like it might only apply to the macOS install script section, when it applies equally to both methods.

I attempted to clarify that by more concisely recommending it in both cases: [here](https://github.com/ddev/ddev/blob/7e5643e81646becf20bbf3434bf848d89f224898/docs/content/users/install/ddev-installation.md?plain=1#L20-L21) and [here](https://github.com/ddev/ddev/blob/7e5643e81646becf20bbf3434bf848d89f224898/docs/content/users/install/ddev-installation.md?plain=1#L32-L33).

I also deliberately left it out of the Linux install script instructions [here](https://github.com/ddev/ddev/blob/7e5643e81646becf20bbf3434bf848d89f224898/docs/content/users/install/ddev-installation.md?plain=1#L116-L117), under the assumption that Mutagen [doesn’t offer a significant enough benefit on Linux](https://ddev.readthedocs.io/en/stable/users/install/performance/#what-mutagen-does).

Am I missing your point?

Reply:
OK, if it's being recommended elsewhere I'm good, just couldn't understand in the diff. 

And yes, it is not normally used/useful in Linux or WSL2, although does no harm there.

Reply:
With this PR we now speak directly and specifically to the Linux user instead of punting to the macOS section, so it felt right to remove that part and align with the Linux notes on the Mutagen page.

Sounds like we’re all good on this front!